broker/consumer/vm_install.sh

```diff
@@ -74,4 +74,4 @@ startupscript="gs://${broker_bucket}/consumer/vm_startup.sh"
 gcloud compute instances add-metadata "$consumerVM" --zone "$zone" \
     --metadata startup-script-url="$startupscript"
 echo "vm_install.sh is complete. Shutting down."
-shutdown -h now
+shutdown -h now
```

Comment:
I previously edited this script and pushed the changes to this PR. Eventually, I decided to discard those changes by replacing the changed code with the original code. Because of this, Github is marking the last line of this file and `broker/consumer/vm_startup.sh` as being changed (I did the same thing with `broker/consumer/vm_startup.sh`), despite the code being identical. I haven't figured out a way to resolve this

Reply:
It's because the `master` has a line feed character at the end, and the changed version doesn't.  The correct answer is that it should have a line feed character.

Reply:
The red circle with the line through it is GitHubs way of telling you this.

Reply:
You will want to configure your default editor to make sure there is always a line feed character at the end of your files.